examples/java/ceph-s3-upload/README.md

```diff
@@ -0,0 +1,16 @@
+# Introduction
+This directory contains Java code examples on how to upload an object to an S3 bucket running on a Ceph RGW cluster.
+
+# Prerequisite
+Linux machine running a RGW Ceph cluster with Java and Maven installed.
+
+## Workflow Procedure
+1. Install AWS CLI version one on your Linux machine as explained [here](https://docs.aws.amazon.com/cli/v1/userguide/install-linux.html)
+2. Once successful, run ```aws configure ``` on your terminal to enter the Ceph RGW cluster access key and secret key into the global shared configuration file.
+3. Create a bucket on the Ceph cluster with the command 
+```
+aws --endpoint-url http://localhost:8000 s3 mb s3://sample-bucket
+```
+4. Navigate through your file system into the ceph-s3-upload folder  using your terminal. Please ensure you see the pom.xml file.
```

Comment:
Wrap `ceph-s3-upload` in double backticks.  The triple backticks elsewhere here should probably be double just for consistency.

Reply:
All done